### DISCUSSION TITLE
slot not found?

### DISCUSSION THREAD

**Discussion by junhee424** [RogueEdit/onlineRogueEditor#109] [Q&A] [answered]
I can successfully login through both methods and input commands but when I try to use 26 to save and upload the data is says "slot_1.json not found"? I checked my account I purposely only have save slot 1 with a fresh run so what's up with that? How do I fix this? Thanks in advance

**Comment by JulianStiebler** [comment, marked as answer]
You could login, but you didnt read what else was going on... You would have cleary seen that it could not fetch your slot-data and therefore it didnt existed, logically.

Its insane to me how people do not read anything anymore.
